src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/36.diff.sql

```diff
@@ -0,0 +1,136 @@
+﻿SET XACT_ABORT ON
+
+BEGIN TRANSACTION
+GO
+
+/*************************************************************
+    File Property Table
+    Stores file properties of a given instance
+**************************************************************/
+IF NOT EXISTS (
+    SELECT *
+    FROM sys.tables
+    WHERE name = 'FileProperty')
+BEGIN
+CREATE TABLE dbo.FileProperty (
+                                           InstanceKey             BIGINT             NOT NULL, --FK
+                                           FilePath                NVARCHAR(4000)     NOT NULL,
+                                           ETag                    NVARCHAR(200)      NOT NULL
+)
+WITH (DATA_COMPRESSION = PAGE)
+END
+
+IF NOT EXISTS (
```

Comment:
Ideally you will create indexes outside of transaction since the table will be locked. In this case it is fine since there is no data and its new one